src/command_modules/azure-cli-batch/HISTORY.rst

```diff
@@ -3,6 +3,17 @@
 Release History
 ===============
 
+4.0.0
++++++
+* Updated to Batch SDK 6.0.0
+    * **[Breaking]** A new property `HttpUrl` replaces `BlobSource`. This can be any HTTP URL. Previously, this had to be an Azure Blob Storage URL.
```

Comment:
I think we had better specify the affected commands.

Reply:
This actually entirely under the hood removing from changelog minus help text

Reply:
Removing from readme